pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.4</version>
+		<relativePath/>
+	</parent>
+	<groupId>com.nowoczesnyjunior</groupId>
+	<artifactId>financial-app</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>financial-app</name>
+	<description>Financial App</description>
+	<properties>
+		<java.version>17</java.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+<!--		<dependency>-->
+<!--			<groupId>org.springframework.boot</groupId>-->
+<!--			<artifactId>spring-boot-starter-security</artifactId>-->
+<!--		</dependency>-->
```

Comment:
It doesn't matter that much, yet deleted code is better than code commented out :)

And there should be possibility to add spring security now and disable autoconfiguration.

I see next task with user auth related to security. 

so for now I would suggest either:
- adding word of explanation why this is commented out
- removing this completely
- actually adding spring security and disabling default config

Reply:
thanks, I choose removing unused dependencies - I will add it later, when I will implement auth